go/libraries/doltcore/mvdata/data_mover.go

```diff
@@ -144,20 +164,56 @@ func (imp *DataMover) Move(ctx context.Context) (badRowCount int64, err error) {
 	return badCount, nil
 }
 
+// writeBadRowToCli prints a bad row in a csv form to STDERR.
+func writeBadRowToCli(r row.Row, sch schema.Schema, b *bytes.Buffer) error {
+	sqlRow, err := sqlutil.DoltRowToSqlRow(r, sch)
+	if err != nil {
+		return err
+	}
+
+	wr := bufio.NewWriter(b)
+
+	colValStrs := make([]*string, len(sqlRow))
+
+	for colNum, col := range sqlRow {
+		if col != nil {
+			str := sqlutil.SqlColToStr(col)
+			colValStrs[colNum] = &str
+		} else {
+			colValStrs[colNum] = nil
+		}
+	}
+
+	err = csv.WriteCSVRow(wr, colValStrs, ",", false)
+	if err != nil {
+		return err
+	}
+
+	err = wr.Flush()
+	if err != nil {
+		return err
+	}
+
+	str := b.String()
+	cli.PrintErr(str)
+
+	return nil
+}
+
 func MoveDataToRoot(ctx context.Context, mover *DataMover, mvOpts DataMoverOptions, root *doltdb.RootValue, updateRoot func(c context.Context, r *doltdb.RootValue) error) (*doltdb.RootValue, int64, errhand.VerboseError) {
 	var badCount int64
 	var err error
 	newRoot := &doltdb.RootValue{}
 
-	badCount, err = mover.Move(ctx)
+	badCount, err = mover.Move(ctx, mover.Wr.GetSchema())
```

Comment:
Still need this extra param? Probably shouldn't, right?

Reply:
No it's needed to print out the actual row.